README.md

```diff
@@ -96,6 +96,31 @@ teams:
       project-groups:
         - <project group>
         - <project group>
+
+    # (Optional) Gitea configuration for the team
+    gitea:
+      # (Required) The name of a Gitea Organisation
+      - org: <gitea organisation name>
+
+        # (Required) The URL of the Gitea instance, e.g. `https://opendev.org/`
+        url: <URL for the Gitea instance>
+
+        # (Optional) A list of repository names to include
+        # If empty, all public, non-archived repositories in the org will be
+        # included, unless listed in `ignores`.
+        includes:
+          # List of repo names
+          - <repo>:
+            monorepo-folders: ['<folder name>', <'folder name'>]
```

Comment:
Because releasegen is about tracking package releases, I wonder if the following might be more explicit:

```yaml
# Long form
- name: some-repo
  package-dirs: ["dir1", "dir2", "dir3/dir4"]

# Short form
- some-other-repo
- a-n-other-repo
```

Thoughts/

Reply:
Yes, that looks better to me. I'll try to figure out how to get viper to do that :laughing:.